src/main/java/stepsdesign/ControlCenterSteps.java

```diff
@@ -0,0 +1,205 @@
+
+  @Then("Click on the schema link of the dataset entity details page")
+  public void clickOnSchemaTabLink() {
+    CdfControlCenterActions.clickOnSchemaTabDataset(); }
+
+  @Then("Verify user is navigated to the schema details page of the dataset entity page")
+  public void navigateToSchemaTabLinkPage() {
+    CdfControlCenterActions.navigateToSchemaTabPage(); }
+
+  @Then("Click on the programs link of the dataset entity details page")
+  public void clickOnProgramsTabLink() {
+    CdfControlCenterActions.clickOnProgramsTabDataset(); }
+
+  @Then("Verify user is navigated to the programs details page of the dataset entity page")
+  public void navigateToProgramsTabLinkPage() {
+    CdfControlCenterActions.navigateToProgramsTabPage(); }
+
+  @Then("Select the sort by dropdown with option value: {string}")
+  public void selectDropdownSortOptionValue(String option) {
+    CdfControlCenterActions.selectSortDropdown();
+    CdfControlCenterActions.selectSortDropdownOptionValue(option); }
+
+  @Then("Verify user is able to see the entities sorted by the newest option")
+  public void clickOnNewestOptionMessage() {
+    CdfControlCenterActions.clickOnNewestOptionMessage(); }
+
+  @Then("Verify user is able to see the entities sorted by the oldest option")
+  public void clickOnOldestOptionMessage() {
+    CdfControlCenterActions.clickOnOldestOptionMessage(); }
+
+  @Then("Verify user is able to see the entities sorted by the Z to A option")
+  public void clickOnZToAOptionMessage() {
+    CdfControlCenterActions.clickOnZToAOptionMessage(); }
+
+  @Then("Verify user is able to see the entities sorted by the A to Z option")
+  public void clickOnAToZOptionMessage() {
+    CdfControlCenterActions.clickOnAToZOptionMessage(); }
+
+  @Then("Click on the plus add button of the dataset entity to add the tags successfully")
+  public void clickOnAddTagIcon() {
+    CdfControlCenterActions.clickOnAddTagIcon(); }
+
+  @Then("Verify user is able to enter the values in tag input field: {string}")
+  public void enterValueInTagInputField(String value) {
+    CdfControlCenterActions.enterValueTagInputField(value);
+    CdfControlCenterActions.pressEnterKey();
+  }
+
+  @Then("Verify that the tag count of dataset entity increases when the user adds the tag")
+  public void tagCountIncreasesDataset() {
+    CdfControlCenterActions.tagCountIncreasesDataset(); }
+
+  @Then("Click on the close icon of the added tag and verify that the tag count decreases")
+  public void clickOnCloseIconAndTagCountDecreases() {
+    CdfControlCenterActions.clickOnCloseIcon(); {
+      CdfControlCenterActions.tagCountDecreasesDataset(); }
+  }
+
+  @Then("Enter the text in search tab {string} in control center")
+  public static void openSearchControlCenter(String value) {
+    CdfControlCenterActions.clickOnSearchTagControlCenter(value);
+  }
+
+  @Then("Verify user can see the searched tag successfully on the control center page")
```

Comment:
Verify the searched tag is displayed successfully on control center page

Reply:
changed

Reply:
Hi Priya , We tried looking for the changed line provided by Surya but it is not seen in any of thecdap repo feature files . If it is not changed there then how the scenario will work ?